test/NuGet.Core.FuncTests/NuGet.XPlat.FuncTest/DotnetCliUtil.cs

```diff
@@ -35,7 +35,7 @@ static DotnetCliUtil()
 #else
                 "Release",
 #endif
-                "netcoreapp1.0",
+                "netcoreapp2.1",
```

Comment:
We should ideally make this configurable. 
Do we have a task to make the tests run with multiple versions of the SDK?

Reply:
We do not have any task for that